src/cache_cf.cc

```diff
@@ -1120,9 +1120,9 @@ CheckTimeValue(const double value, const std::chrono::nanoseconds &unit)
     if (value < 0)
         throw TexcHere("time must have a positive value");
 
-    const auto maxNanoseconds = std::chrono::nanoseconds::max().count();
-    if (value > maxNanoseconds/static_cast<double>(unit.count())) {
-        const auto maxYears = maxNanoseconds/(HoursPerYear*3600*1000000000);
+    if (unit > std::chrono::nanoseconds::max()) {
+        // XXX std::chrono::years is c++20
```

Comment:
Nit: This is not really an XXX (nothing is really broken in this code) but a TODO. _And_ there is already a TODO about this problem, in the place that is relevant to this and any similar code:

```C++
// std::chrono::years requires C++20. Do our own rough calculation for now.
static const double HoursPerYear = 24*365.2522;
```

I wish the above had a TODO keyword, but at least it will match `C++20` when we will start that migration.

Thus, I suggest removing this XXX comment:

```suggestion
```

Reply:
OK